### PR TITLE
Create limited AWS IAM user for CI deploys

### PR DESCRIPTION
**Pull request by vkurup**
I've tested this out on PW. I did have to turn `k8s_rollout_after_deploy` off (Should I mention that here in the docs?)

### CODE REVIEW

File: templates/aws/ECRPush.json.j2 [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+   "Version":"2012-10-17",
+   "Statement":[
+     {% if k8s_ci_vault_password_arn %}
```
Comment (copelco) [inline templates/aws/ECRPush.json.j2:4]: Cool!

**Review by copelco — APPROVED** [review]
Thanks for adding this @vkurup! :shipit:

**Comment by copelco** [comment]
Oh I missed your comment re ``k8s_rollout_after_deploy``. Does rollout only fail with CI?

**Comment by vkurup** [comment]
Yes, rollout succeeded with my AWS Admin user on my laptop, but failed in CI, so I think it takes more permissions to run kubectl in an ansible command, then it does to apply the k8s ansible roles.

**Comment by vkurup** [comment]
... which makes sense to me now because those tasks don't use the k8s_auth_api_key, which is the credential for the service account. I wonder if there is a way to do use those credentials with kubectl. I'll create a new issue to research that when someone has time.

**Pull request merged**